basin3d/synthesis.py

```diff
@@ -691,6 +702,11 @@ def _output_df(output_directory, output_name, query_info, metadata_store, first_
     metadata_data_df = _fill_metadata_df(metadata_store, synthesized_var_list)
     metadata_nodata_df = _fill_metadata_df(metadata_store, synthesized_no_var_list)
 
+    # Only Drop NaNs if temporal resolution is < 'DAY'
+    if temporal_resolution in ['MINUTE', 'HOUR', 'SECOND']:
```

Comment:
Seems like `NONE` should be added here. Or a note in the comment that `NONE` needs to be handled in future reworks.

Reply:
@dschristianson thanks for pointing this out.  Since this PR is before @heycatwonton's instantaneous PR, I left it out until she implements NONE.  This PR doesn't "know" anything about the NONE option.